meta/store.go

```diff
@@ -980,11 +984,18 @@ func (s *Store) Authenticate(username, password string) (ui *UserInfo, err error
 			return ErrUserNotFound
 		}
 
+		// Check the local auth cache first.
+		if p, ok := s.authCache[username]; ok && p == password {
```

Comment:
should change this. if it's `ok` but `p != password` we should return an authentication error rather than doing the bcrypt. Means that someone could easily DDOS an InfluxDB server with bad auth info